test/E2ETest/Microsoft.Test.E2E.AspNet.OData/BulkOperation/BulkOperationController.cs

```diff
@@ -249,15 +249,16 @@ public ITestActionResult PatchEmployees([FromBody] DeltaSet<Employee> coll)
 
         [ODataRoute("Employees")]
         [HttpPost]
+        [EnableQuery]
         public ITestActionResult Post([FromBody] Employee employee)
         {
-            InitEmployees();
+            //InitEmployees();
 
-            var handler = new EmployeeAPIHandler();
+            //var handler = new EmployeeAPIHandler();
 
-            handler.UpdateLinkedObjects(employee, Request.GetModel());
+            //handler.UpdateLinkedObjects(employee, Request.GetModel());
```

Comment:
Why are these commented out?

Reply:
They are currently not being used. They will be activated in the Deep insert PR